graphql-java-servlet/src/main/java/graphql/kickstart/servlet/BatchedQueryResponseWriter.java

```diff
@@ -34,6 +41,16 @@ public void write(HttpServletRequest request, HttpServletResponse response) thro
 
     String responseContent = responseBuilder.toString();
     byte[] contentBytes = responseContent.getBytes(StandardCharsets.UTF_8);
+
+    if (responseCache != null && responseCache.isCacheable(request, invocationInput)) {
+      try {
+        responseCache.put(request, invocationInput, CachedResponse.ofContent(contentBytes));
+      } catch (Throwable t) {
+        log.warn(t.getMessage(), t);
```

Comment:
I'd loose this log statement and just add the stacktrace to the log statement on the next line.

Reply:
Done.